File: Data Structures & Algorithms/sliding-window-maximum/submission-0.java
class Solution {
    public int[] maxSlidingWindow(int[] nums, int k) {
        int[] res = new int[nums.length - k + 1];
        for (int r = 0; r < nums.length - k + 1; r++) {
            int max = nums[r];
            for (int s = r + 1; s < r + k; s++) {
                if (nums[s] > max) {
                    max = nums[s];
                }   
            }
            res[r] = max;
        }
        return res;
    }
}

File: Data Structures & Algorithms/subsets/submission-0.py
import copy
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res = [[]]

        for num in nums:
            current = copy.deepcopy(res)
            for i in range(len(current)):
                current[i].append(num)
                res.append(current[i])
        return res